css/types/image.json

```diff
@@ -105,6 +105,306 @@
+                      "Considers <code>&lt;angle&gt;</code> to start to the right, instead of the top. I.e. it considered an angle of <code>0deg</code> as a direction indicator pointing to the right."
+                    ]
+                  }
+                ],
+                "safari_ios": {
+                  "version_added": true
+                },
+                "samsunginternet_android": {
+                  "version_added": true
+                }
+              },
+              "status": {
+                "experimental": false,
+                "standard_track": true,
+                "deprecated": false
+              }
+            },
+            "to": {
+              "__compat": {
+                "description": "<code>to</code> keyword",
+                "support": {
+                  "webview_android": {
+                    "version_added": true
+                  },
+                  "chrome": {
+                    "version_added": "26"
+                  },
+                  "chrome_android": {
+                    "version_added": true
+                  },
+                  "edge": {
+                    "version_added": "12"
+                  },
+                  "edge_mobile": {
+                    "version_added": true
+                  },
+                  "firefox": {
+                    "version_added": "10"
+                  },
+                  "firefox_android": {
+                    "version_added": "10"
+                  },
+                  "ie": {
+                    "version_added": "10"
+                  },
+                  "opera": {
+                    "version_added": "12.1"
+                  },
+                  "opera_android": {
+                    "version_added": true
+                  },
+                  "safari": {
+                    "version_added": "6.1"
+                  },
+                  "safari_ios": {
+                    "version_added": true
+                  },
+                  "samsunginternet_android": {
+                    "version_added": true
+                  }
+                },
+                "status": {
+                  "experimental": false,
+                  "standard_track": true,
+                  "deprecated": false
+                }
+              }
+            },
+            "interpolation_hints": {
+              "__compat": {
+                "description": "Interpolation Hints / Gradient Midpoints",
+                "support": {
+                  "webview_android": {
+                    "version_added": true
```

Comment:
This and chrome_android should also be 40.

Reply:
Done